starter_code/lab-express-cinema/routes/index.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const router  = express.Router();
+
+/* GET home page */
+router.get('/', (req, res, next) => {
+  res.render('index');
+});
+
+router.get('movie-detail/:id'), (req, res, next) => {
+  let movieId = req.params.id;
```

Comment:
tenéis un par de errores en esta línea, falta / en la ruta y sobre el paréntesis de cierre después de la ruta, sería así:
```
router.get('/movie-detail/:id', (req, res, next) => {